### PR TITLE
OBJ-101 error handling for invalid file types

### PR DESCRIPTION
**Pull request by mattch1**
Error message code to handle 415 status error from the api.

### CODE REVIEW

File: src/controllers/document_upload/document.upload.controller.ts [hunk 1]
```diff
@@ -157,10 +159,10 @@ const getUploadFinishedCallback = (req: Request,
 
       // TODO OBJ-101 - file type error - uncomment below should do it
```
Comment (mwestacott) [inline src/controllers/document_upload/document.upload.controller.ts:160]: Guessing that TODO comment should now be deleted if this is the full and final implementation of OBJ-101.

File: src/controllers/document_upload/document.upload.controller.ts [hunk 2]
```diff
@@ -157,10 +159,10 @@ const getUploadFinishedCallback = (req: Request,
...
       // // render errors in the view
-      // if (e.status === 415) {
-      //   return await
-      //     displayError(req, res, UploadErrorMessages.INVALID_MIME_TYPES, uploadResponderStrategy, attachments);
-      // }
+      if (e.status === 415) {
```
Comment (mwestacott) [inline src/controllers/document_upload/document.upload.controller.ts:162]: 415 is a magic number! Would be good to find out what this error code actually means, define a well-named constant for it and use that.

Comment (mattch1) [reply src/controllers/document_upload/document.upload.controller.ts:162]: Not sure what you mean by magic number - but will extract a constant if there are no references to this.

Comment (mwestacott) [reply src/controllers/document_upload/document.upload.controller.ts:162]: https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

And thanks.

File: test/controller/document_upload/document.upload.controller.spec.unit.ts [hunk 3]
```diff
@@ -374,7 +374,7 @@ describe ("document.document_upload.controller tests", () => {
       throw {
         data: ["error"],
         message: "error",
-        status: 415,
+        status: HttpStatusCodes.UNSUPPORTED_MEDIA_TYPE,
```
Comment (markpit) [inline test/controller/document_upload/document.upload.controller.spec.unit.ts:377]: could you change this back to 415, the real error code will be a number , just want to make sure that when we do the comparison in the code, it works when comparing the number to the enum.

Comment (markpit) [reply test/controller/document_upload/document.upload.controller.spec.unit.ts:377]: just in the test i mean

File: src/controllers/document_upload/document.upload.controller.ts (hunk 2 above)
Comment (mattch1) [reply src/controllers/document_upload/document.upload.controller.ts]: Updated this.

File: src/controllers/document_upload/document.upload.controller.ts (hunk 1 above)
Comment (mattch1) [reply src/controllers/document_upload/document.upload.controller.ts]: Removed the TODO

File: test/controller/document_upload/document.upload.controller.spec.unit.ts (hunk 3 above)
Comment (mattch1) [reply test/controller/document_upload/document.upload.controller.spec.unit.ts]: Ok chnaged back to "415".

**Pull request merged**